app/views/events/index.html.slim

```diff
@@ -5,7 +5,7 @@
     form
       input.form-control type="text" name="search" placeholder="Search for events, shows, courses..."
 .container-fluid.events-list
-  - @events.in_groups_of(3) do |group|
+  - @events.in_groups_of(2) do |group|
     .row
       - group.compact.each do |event|
         = render partial: "card", locals: { event: event }
```

Comment:
This is also a N+1 case :D Please take note for me for all these so we can revisit in the end. It's quite advanced.